xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -117,6 +117,7 @@ SelectionStream& CSelectionStreams::Get(StreamType type, int index)
 
 std::vector<SelectionStream> CSelectionStreams::Get(StreamType type)
 {
+  CSingleLock lock(m_section);
```

Comment:
What does that mean? What is "other thread"? Does that mean the class doesn't need protection at all, and the mutex should be removed? There is protection currently, and it's incorrect. Either remove it completely, or fix it (like I did).